packages/ui-components/src/ExpandableText/ExpandableText.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import styles from "./ExpandableText.module.scss";
+import classNames from "classnames/bind";
+
+interface ExpandableTextProps {
+  text: string;
+  characterLimit: number;
+}
+
+const cx = classNames.bind(styles);
+
+export function ExpandableText({
+  text,
+  characterLimit,
+}: ExpandableTextProps): React.ReactElement {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  useEffect(() => {
+    setIsExpanded(false);
+  }, [text]);
+
+  if (text.length <= characterLimit) return <>{text}</>;
+
+  return (
+    <>
+      {!isExpanded && (
+        <>
+          {text.slice(0, characterLimit) + "..."}{" "}
```

Comment:
nit: any chance we could use an ellipsis character (…) instead of three periods?